bin/compile

```diff
@@ -178,43 +195,31 @@ if [ "$GIT_SSH_KEY" != "" ]; then
   ssh -oStrictHostKeyChecking=no -T git@github.com || true
 fi
 
+status "Pruning cached dependencies not specified in package.json"
+npm prune 2>&1 | indent
+
+status "Pruning cached bower dependencies not specified in bower.json"
+bower prune 2>&1 | indent
```

Comment:
Hi @moha297,

Can you try to deploy using:
- `heroku config:set REBUILD_ALL=true`

Then try an other deployment without:
- `heroku config:unset REBUILD_ALL` 

If you keep having this issue I'll need the full log, for this can you switch your buildpack to:
- `heroku buildpacks:set https://github.com/GabKlein/heroku-buildpack-ember-cli.git`